### ISSUE TITLE
DNS unreachable

### ISSUE THREAD

**Issue by vrutkovs** [vrutkovs/do-ocp-4.1#3]
`certs` container on the hosts loops in:
```
error sending CSR to signer: Post https://api-int.ocp.vrutkovs.eu:6443/apis/certificates.k8s.io/v1beta1/certificatesigningrequests: dial tcp: lookup api-int.ocp.vrutkovs.eu on 67.207.67.2:53: no such host
```

**Comment by vrutkovs** [comment]
`/etc/resolve.conf` has entries to internal DNS servers, but since private IPs are not enable these resolvers are unreachable.

**Comment by vrutkovs** [comment]
Fixed in adfa08d863ffd4c9910b48ed6d3a4fe406c0d7f6
